src/workerd/util/sqlite.h

```diff
@@ -51,13 +50,57 @@ class SqliteDatabase {
   // expected.
   operator sqlite3*() { return db; }
 
+// Class which regulates a SQL query, especially to control how queries created in JavaScript
+// application code are handled.
+class Regulator {
```

Comment:
The compiler insisted on Regulator being defined before the static constexpr Regulator variable, so I moved the subclass inside.